airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoDatabase.java

```diff
@@ -47,6 +49,9 @@ public MongoDatabase(final String connectionString, final String databaseName) {
       this.connectionString = new ConnectionString(connectionString);
       mongoClient = MongoClients.create(this.connectionString);
       database = mongoClient.getDatabase(databaseName);
+    } catch (final MongoConfigurationException e) {
+      LOGGER.error(e.getMessage());
```

Comment:
Here and other places. I think it would be make failures easier to debug if we always passed the exception object to `LOGGER.error`, so that we can use Log4J configs to also log stack traces. E.g.:
```
LOGGER.error(e.getMessage(), e);
```
or
```
LOGGER.error("Airbyte specific error message", e.getMessage(), e);
```